components/builder-web/stylesheets/base/_variables.scss

```diff
@@ -27,7 +27,7 @@ $base-line-height: 1.5;
 $heading-line-height: 1.2;
 
 // Other Sizes
-$base-border-radius: 7px;
+$base-border-radius: 2px;
```

Comment:
This setting is picked up and propagated app-wide by `$global-radius`, so it'll change a bunch of stuff; just wanted to make sure that's the intention.

Reply:
yeah @ryankeairns updated the mocks to have a 2px rounding

Reply:
We're going more square!

Reply:
square corners are the new rounded corners